common/app/conf/switches.scala

```diff
@@ -640,7 +640,7 @@ object Switches {
     "international-edition",
     "International edition A/B test on",
     safeState = Off,
-    sellByDate = new LocalDate(2015, 7, 31),
+    sellByDate = new LocalDate(2015, 9, 30),
```

Comment:
setting switches to a random day in the month is better than a special date as we always get a flurry of switches expiring at the end of the month